src/main/java/org/folio/circulation/domain/ConfigurationRepository.java

```diff
@@ -1,17 +1,25 @@
 package org.folio.circulation.domain;
 
-import static org.folio.circulation.domain.MultipleRecords.from;
-import static org.folio.circulation.support.CqlQuery.exactMatch;
-
-import java.util.concurrent.CompletableFuture;
-
 import org.folio.circulation.support.Clients;
 import org.folio.circulation.support.CollectionResourceClient;
 import org.folio.circulation.support.CqlQuery;
 import org.folio.circulation.support.Result;
 import org.joda.time.DateTimeZone;
 
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Function;
+
+import static org.folio.circulation.domain.MultipleRecords.from;
```

Comment:
We tend to use the [Eclipse import ordering](https://issues.folio.org/browse/FOLIO-1667), which defines that static imports are imported before non-static, please update this to reflect that

Reply:
Fix